modules/react/text/lib/LabelText.tsx

```diff
@@ -1,18 +1,57 @@
 import * as React from 'react';
 import {Property} from 'csstype';
 import {createComponent} from '@workday/canvas-kit-react/common';
-import {Text, TextProps} from './Text';
-import {inputColors} from '@workday/canvas-kit-react/tokens';
+import {base, system} from '@workday/canvas-tokens-web';
+import {CSProps, createStencil} from '@workday/canvas-kit-styling';
+import {CommonStyleProps, mergeStyles} from '@workday/canvas-kit-react/layout';
 
-export interface TypeLabelProps extends TextProps {
+export interface TypeLabelProps extends CSProps, CommonStyleProps {
   cursor?: Property.Cursor;
   disabled?: boolean;
+  /**
+   * Type variant token names: `error`, `hint` or `inverse`.
+   *
+   * ```tsx
+   * <LabelText variant="error">Error text</LabelText>
+   * ```
+   */
+  variant?: 'error' | 'hint' | 'inverse';
```

Comment:
should this include `undefined` isn't it possible to not have a variant?

Reply:
it's already marked as not required by `?`, so it's possible to be `undefined`